Resources/Textures/ADT/Objects/Weapons/Melee/baseball_bat.rsi/meta.json

```
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA-3.0",
+  "copyright": "Taken from goonstation and modified by Swept at commit https://github.com/goonstation/goonstation/pull/3555/commits/b24eb6260647c0fcfe858268a26b6160bc50017a, wielded version by Easypoll, vertical icon sprite by TiniestShark",
```

Biome (2.5.1)

[error] The key copyright was already declared.

(lint/suspicious/noDuplicateObjectKeys)